### PR TITLE
Change to open telemetry standard

### PR DESCRIPTION
**Pull request by thomas-siegrist**

### CODE REVIEW

File: eventdriven/principles.md [hunk 1]
```diff
@@ -58,6 +58,26 @@ Use certificates or infrastructure users for the connection between machines (e.
 forward the real users token (OAuth token) in the message header. The OAuth token is only required for internal communication
 where the consuming system has to check the real users permissions / roles.
 
+## Monitoring
+
+### `MUST` Support OpenTelemetry
+
+Distributed Tracing over multiple applications, teams and even larg solutions, is very important in root cause analysis and helps detect how latencies are stacked up and where incidents are located and thus can significantly shorten _mean time to repair_.
```
Comment (kguelseven) [inline eventdriven/principles.md:65]: large, drop the second comma

File: eventdriven/principles.md [hunk 2]
```diff
@@ -58,6 +58,26 @@ Use certificates or infrastructure users for the connection between machines (e.
...
+
+To identify a specific request throug the entire chain and beyond team boundaries, every team (and API) `MUST` use (OpenTelemetry)[https://opentelemetry.io/] as it's way to trace calls and business transactions. Teams `MUST` use standard (W3C Trace Context)[https://www.w3.org/TR/trace-context/] Headers, as they are the common standard for distributed tracing and are supported by most of the cloud platforms and monitoring tools. We explicitly use W3C standards for eventing too and do not differ between synchronous and asynchronous requests, as we want to be able to see traces overarching the boundaries of these two architectural patterns.
```
Comment (kguelseven) [inline eventdriven/principles.md:67]: through, drop comma, its (not it's), overarching? (crossing?)

**Review by kguelseven — COMMENTED** [review]
Only typos :)

File: eventdriven/principles.md (hunk 2 above)
Comment (thomas-siegrist) [reply eventdriven/principles.md:67]: thanks Kori!

**Pull request merged**